app/models/sbv/group.rb

```diff
@@ -78,7 +82,7 @@ def song_counts
     SongCount.joins(:concert).where("concerts.verein_id IN (#{verein_sql})")
   end
 
-  def recognized_members
+  def calculated_recognized_members
```

Comment:
I know ... recognized_members was not introduced by you ... but thy not just call this members_count ?